### PR TITLE
Add function to remove signer and pauser & add state variable previousSmartContract

### PR DESCRIPTION
**Pull request by NicolasMahe**
Closes https://github.com/mesg-foundation/atd/issues/3

This PR add 2 new functions: `removeSigner` and `removePauser`. Both function can only be called by the Owner. Those functions allow the owner to remove signers and pausers if for some reason their account is lost or hacked.

### CODE REVIEW

File: contracts/ATD.sol [hunk 1]
```diff
@@ -15,14 +16,18 @@ contract ATD is Pausable, SignerRole, DistributionConfigurable {
 
   ERC20Detailed public token;
 
+  address previousSmartContract;
```
Comment (krhubert) [inline contracts/ATD.sol]: What for is this variable?

If you want to keep previous version of smart contracts shouldn't you write contract designed for it (like Migration contract, but for keeping all versions)

Comment (antho1404) [reply contracts/ATD.sol]: that's a good point, maybe we just need a register of these ATD smart contracts, that might be nicer with a fromblock, toblock period and so on.
Agree to remove that 👍

Comment (antho1404) [reply contracts/ATD.sol]: @NicolasMahe ?

Comment (NicolasMahe) [reply contracts/ATD.sol]: Yes I agree to use a registar rather than a chained of previous contract. I started to do it but it's not done yet.

Comment (antho1404) [reply contracts/ATD.sol]: created a PR to remove that part https://github.com/mesg-foundation/atd/pull/10

**Comment by NicolasMahe** [comment]
@krhubert @antho1404 ready to review

**Comment by antho1404** [comment]
@NicolasMahe ready to merge and deploy, let's try to have this before monday

**Pull request merged**